src/main/java/org/kainos/ea/api/JobService.java

```diff
@@ -28,5 +30,30 @@ public List<Job> getAllJobs() throws FailedToGetJobsException {
 
     }
 
+    public Job getById(int id) throws FailedToGetJobsException {
+        try {
+            return jobDao.getJobById(databaseConnector.getConnection(),id);
+        } catch (SQLException e) {
+            System.err.println(e.getMessage());
+            throw new FailedToGetJobsException();
+        }
+    }
+
+    public boolean delete(int id) throws FailedtoDeleteException {
```

Comment:
don't need to return boolean, just use void

Reply:
Converted to use exceptions